package.json

```diff
@@ -69,27 +70,18 @@
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "eslint-plugin-tailwindcss": "^3.13.0",
     "eslint-plugin-unused-imports": "^3.0.0",
+    "eslint": "^8.48.0",
     "next-transpile-modules": "^10.0.1",
-    "postcss": "^8.4.29",
     "postcss-import": "^15.1.0",
     "postcss-nesting": "^12.0.1",
+    "postcss": "^8.4.29",
     "sass": "^1.66.1",
-    "sharp": "^0.32.5",
+    "tailwindcss": "^3.3.3",
     "ts-loader": "^9.4.4",
     "typescript": "^5.2.2",
     "vitest": "^0.34.3",
     "webpack": "^5.88.2"
   },
-  "resolutions": {
-    "bn.js": "5.2.1",
-    "babel-loader": "9.1.0",
-    "framer-motion": "7.6.15",
-    "ethereumjs-util": "6.1.0",
-    "schema-utils": "4.0.0",
-    "tr46": "1.0.1",
-    "d3-color": "^3.1.0",
-    "json5": ">=2.2.2"
-  },
```

Comment:
@Majorfi can you please double check that these are not needed? I removed all the resolutions, and AFAIK they don't seem to have any impact. Were these legacy ones or do you have some more context here?